packages/utils/parse.ts

```diff
@@ -75,6 +89,28 @@ export const configuredXss = new FilterXSS({
       }
       return `${name}="${escapeAttrValue(allowedClasses.join(' '))}"`
     }
+
+    // For markdown callouts
+    if (name === 'class' && ['div', 'p'].includes(tag)) {
```

Comment:
again it seems weird that this is being whitelisted within the input, as someone could put these classes on their own html elements and have weird styles happen